api/app/wildfire_one/schema_parsers.py

```diff
@@ -130,20 +131,50 @@ def generate_station_daily(raw_daily, station: WFWXWeatherStation, fuel_type: st
     pc = FUEL_TYPE_LOOKUP[fuel_type]["PC"]
     pdf = FUEL_TYPE_LOOKUP[fuel_type]["PDF"]
     cbh = FUEL_TYPE_LOOKUP[fuel_type]["CBH"]
+    cfl = FUEL_TYPE_LOOKUP[fuel_type]["CFL"]
 
     isi = raw_daily.get('initialSpreadIndex', None)
     bui = raw_daily.get('buildUpIndex', None)
     ffmc = raw_daily.get('fineFuelMoistureCode', None)
     fmc = cffdrs.foliar_moisture_content(station.lat, station.long, station.elevation, get_julian_date_now())
-    sfc = cffdrs.surface_fuel_consumption(fuel_type, bui, ffmc, pc)
+
+    sfc = None
+    try:
+        sfc = cffdrs.surface_fuel_consumption(fuel_type, bui, ffmc, pc)
+    except cffdrs.CFFDRSException as exception:
+        logger.error(exception, exc_info=True)
 
     cc = raw_daily.get('grasslandCuring')
     if cc is None:
         cc = FUEL_TYPE_LOOKUP[fuel_type]["CC"]
-    ros = cffdrs.rate_of_spread(fuel_type, isi, bui, fmc, sfc, pc=pc,
-                                cc=cc,
-                                pdf=pdf,
-                                cbh=cbh)
+
+    ros = None
+    try:
+        if sfc is not None:
```

Comment:
why are we checking that `sfc` is not None but we don't do checks for `None` on `isi`, `bui`, `ffmc`, `fmc`? Seems like the error handling is inconsistent across variables

Reply:
It's because this is the only variable that breaks the call to the R library. I'm basically trusting the `cffdrs` wrapper error handling verbatim.

Reply:
I'll double check the `hfi` block though.